godot_project/editor/controls/dockers/workspace_docker/c_dynamic_context_docker/dynamic_context_controls/inspector_controls/inspector_control_bond_order/inspector_control_bond_order.gd

```diff
@@ -37,6 +37,10 @@ func setup(in_structure_context: StructureContext, in_bond_id: int, in_is_editab
 	var bond: Vector3i = in_structure_context.nano_structure.get_bond(_bond_id)
 	var bond_order: int = bond.z
 	_option_button.select(BOND_ORDER_TO_OPTION_IDX[bond_order])
+	
+	var rendering: Rendering = in_structure_context.get_rendering()
+	var can_be_edited_by_user: bool = rendering.get_default_representation() != Rendering.Representation.STICKS
+	_option_button.disabled = not can_be_edited_by_user
```

Comment:
will this be updated if Rendering.Representation changes after it is created?

Reply:
It will not, Good catch!